Animations/jquery Animations/Examples/scroll to top of body and change color and div onclick fixed menu bar/custom.css

```diff
@@ -0,0 +1,61 @@
+* {
+	margin:0;
+	padding: 0;
+	text-decoration: none;
+	list-style-type: none;
+}
+ul li {
+margin: 30px;
+}
+ul li a {
+color: green;
+}
+.active {
+color: red;
+}
+p {
+font-size: 18px;
+line-height: 25px;
+}
+.leftSection {
+position: fixed;
+}
+.rightSection {
+float: left;
+width: 70%;
+padding-left: 30%;
+}
+.top {
+display: none;
+position: fixed;
+height: 20px;
+width: 20px;
+background-color: #000;
+bottom: 50px;
+right: 50px;
+}
+@media only screen and (max-width:767px) {
+	.leftSection {
+		width: 100%;
+		top: 0;
+	}
+	.rightSection {
+		width: 100%;
+		padding-left: 0;
+		float: none;
+	}
+	ul {
+		background-color: #000;
+	}
+	ul li {
+		float: left;
+	}
+	ul li.clear {
+		float: none;
+		margin: 0;
+		clear: both;
+	}
+	#First,#Second,#Third {
```

Comment:
Why id's and padding-top in css?

Reply:
For mobile view position of menubar fixed so when u click on anchors in menu each div goes to top so padding to each div is given.I don't know its solution it is set by offest .Please give its solution.That coding is faulty as it added more space between divs.